roles/callhome/node/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# handlers file for node
+- name: yum-clean-metadata
+  command: yum clean metadata
```

Comment:
This is very OS specific, and only would work for yum based package managers... not setting up a way to support different package managers in the future (apt, zypper, etc..)